src/styles/index.scss

```diff
@@ -1,3 +1,79 @@
+$stopwatch_height: 80vmin;
+$stopwatch_widht: 80vmin;
+$minutes_height: 20vmin;
+$minutes_widht: 3vmin;
+$seconds_height: 38vmin;
+$seconds_widht: 1.5vmin;
+
+@mixin position {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+}
```

Comment:
you have it already in different files so just import them and delete this repetition